libs/common/src/types/mongo-db-models/tfm-activity.ts

```diff
@@ -0,0 +1,27 @@
+import { ACTIVITY_TYPES } from '../../constants';
+import { UnixTimestamp, UnixTimestampSeconds } from '../date';
+
+export type TfmActivity =
+  | {
+      type: typeof ACTIVITY_TYPES.ACTIVITY | typeof ACTIVITY_TYPES.COMMENT;
```

Comment:
Can we destructure `ACTIVITY_TYPES` in here, I think it makes sense to _not_ do this in the previousf files, but in here it's a tongue twister 😄 

```ts
type: typeof ACTIVITY_TYPES.ACTIVITY | typeof
```